internal/projection/process/event_enricher.go

```diff
@@ -105,9 +105,15 @@ func GetBaseEnrichedEvent(event *types.Event, cluster map[string]interface{}, ho
 
 	enrichedEvent.ID = uuid.NewSHA1(namespace, []byte(enrichedEvent.Message+enrichedEvent.EventTime)).String()
 
-	if versions, ok := event.Metadata["versions"]; ok {
-		if v, ok := versions.(map[string]interface{}); ok {
-			enrichedEvent.Versions = v
+	if versionsInterface, ok := event.Metadata["versions"]; ok {
```

Comment:
To improve readability of this function we could extract this to possibly two methods `getVersionsFromMetadata` `getReleaseTagFromMetadata` or something alike,